tests/test_end_to_end.py

```diff
@@ -172,3 +174,60 @@ def test_reference_grib(
     # various inconsistencies (of dtype casting int to float, etc.). With the right combination of
     # options passed to the pipeline, seems like these should pass?
     # xr.testing.assert_equal(ds.load(), ds2)
+
+
+def test_pyramid(
+    pyramid_datatree,
+    netcdf_local_file_pattern,
+    pipeline,
+    tmp_target,
+):
+    class SetCRS(beam.PTransform):
+        """Updates CRS and coord naming"""
+
+        @staticmethod
+        def _set_CRS(item: Indexed[xr.Dataset]) -> Indexed[xr.Dataset]:
+            index, ds = item
+
+            import rioxarray  # noqa
+
+            ds = ds.rename({"lon": "longitude", "lat": "latitude"})
+            ds.rio.write_crs("epsg:4326", inplace=True)
+            return index, ds
```

Comment:
Is this needed anymore after adding the epsg_code and rename_spatial_dims parameters?